TimerCalculatorUtility/TimerCalculatorUtility/ViewModels/MenuPageViewModel.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Collections.Generic;
+using TimerCalculatorUtility.Views;
+using Xamarin.Forms;
+using MenuItem = TimerCalculatorUtility.Models.MenuItem;
+
+namespace TimerCalculatorUtility.ViewModels
+{
+    public class MenuPageViewModel
```

Comment:
Usually, when you use View models, you need to extend INotifyPropertyChanged.
For you specific case, it would be better if you create BaseViewModel and move shared functionality for all view models into that class, then inherit this class for every other view model